test/mocked_sc_htmlmerger.js

```diff
@@ -14,9 +14,10 @@ connect().use('/sc/htmlmerger', function fooMiddleware(req, res, next) {
         // res.write("Chunk");
         const parts = pair.split('=');
         const appName = parts[0];
-        const fileName = parts[1];
+        let fileName = parts[1];
         content += `<imported-template-scope scope="${appName}"><template><meta itemprop="juicy-composition-scope" content="${appName}"/></template>`;
         if(fileName){
+            fileName = fileName.replace('%2F','/');
             if(fs.existsSync(servingDirectory + fileName)){
```

Comment:
This is usually a bad idea. I would do [`path.resolve(__dirname, 'sc', fileName)`](https://nodejs.org/api/path.html#path_path_resolve_paths).

Reply:
This is not Node, it is a web browser. I propose to use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/decodeURIComponent